microverse_collins_yemi-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="main">
+    <div>
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input placeholder="Enter your email">
+      <h1 class="bgYellow">SELECT</h1>
+    </div>
```

Comment:
HTML best practices state that only one h1 tag should be used on a web page. You have already used the h1 tag twice in this  section. Kindly change this tag to another hx tag between h2-h6 and also try giving your input tag a type attribute.

```suggestion
      <h1>Handcrafted, home-made masterpieces</h1>
      <input type="email" placeholder="Enter your email">
      <h2 class="bgYellow">SELECT</h2>
    </div>
```